Tools/SeeDot/seedot/Predictor/library_fixed.cpp

```diff
@@ -954,6 +954,64 @@ void Maxpool(MYINT *A, MYINT *B, MYINT N, MYINT H, MYINT W, MYINT C, MYINT FH, M
 	return;
 }
 
+// A = Normalise(A)
+void NormaliseL2(MYINT* A, MYINT N, MYINT H, MYINT W, MYINT C, MYINT scaleA, MYINT shrA) {
```

Comment:
name this L2Norm4 (add number of dimensions?)

Reply:
That would mean calculating L2 Norm. But we calculated the norm and then divide the numbers by norm. We can just add the dimension name if in future we add some other dimension.